### PR TITLE
[#821] - Migrar inputs de StoryEditionDateLabelComponent a signals

### PR DESCRIPTION
**Pull request by poojatalele**

### CODE REVIEW

**Review by rolivencia — CHANGES_REQUESTED** [review]
Same as #880: please leave out changes related to `package.json` and `pnpm-lock.yaml` files.

Also you've included changes to `share-content.component.ts`, which shouldn't be here.

**Comment by poojatalele** [comment]
> Same as #880: please leave out changes related to `package.json` and `pnpm-lock.yaml` files.
> 
> Also you've included changes to `share-content.component.ts`, which shouldn't be here.

did the required changes for this PR. please review it.

File: src/app/components/story-edition-date-label/story-edition-date-label.component.ts [hunk 1]
```diff
@@ -27,6 +27,6 @@ import { CommonModule, NgIf } from '@angular/common';
 	imports: [CommonModule, NgIf],
 })
 export class StoryEditionDateLabelComponent {
-	@Input() label: string | undefined;
-	@Input() markAsNew: boolean = false;
+	label = input<string | undefined>();
```
Comment (rolivencia) [inline src/app/components/story-edition-date-label/story-edition-date-label.component.ts:30]: When migrating to signals we do not need to specify `undefined` as a possible type for the inputs when they're not required. Please keep this one as `label = input<string>` and we're good to go.

**Review by rolivencia — CHANGES_REQUESTED** [review]

File: src/app/components/story-edition-date-label/story-edition-date-label.component.ts (hunk 1 above)
Comment (poojatalele) [reply src/app/components/story-edition-date-label/story-edition-date-label.component.ts]: Done!!

**Review by rolivencia — CHANGES_REQUESTED** [review]
It seems we have a compilation problem for this one yet @poojatalele. The build is throwing these errors:

```
▲ [WARNING] NG8109: label is a function and should be invoked: label() [plugin angular-compiler]
    src/app/components/story-edition-date-label/story-edition-date-label.component.ts:10:7:
      10 │         {{ label }}
         ╵            ~~~~~
✘ [ERROR] NG4: This condition will always return true since this function is always defined. Did you mean to call it instead? [plugin angular-compiler]
    src/app/components/story-edition-date-label/story-edition-date-label.component.ts:14:7:
      14 │     @if (markAsNew) {
         ╵          ~~~~~~~~~
```

To fix this, which was caused after we migrated to signals, we must:

1. Make `label` to be called as a function, as `label()`, in the component template in line 10 of `story-edition-date-label.component.ts`
2. Make `markAsNew` to be called as a function, as `markAsNew()`, in the component template in line 14 of `story-edition-date-label.component.ts`

The error happens because a signal is a type of function, so to get the current value of a signal we have to invoke the function itself to get/unwrap its current value.

**Comment by poojatalele** [comment]
pls check now

**Review by rolivencia — APPROVED** [review]
It's good to go now @poojatalele. Thanks for your efforts 🤝

**Pull request merged**